Oxide.Core/RemoteConsole/RemoteConsole.cs

```diff
@@ -76,21 +87,23 @@ private void OnMessage(MessageEventArgs e)
                     var commands = message.Message.Split(' ');
                     covalence.Server.Command(commands[0], commands.Length > 1 ? commands.Skip(1).ToArray() : null);
                     break;
+
                 case "chat":
                     covalence.Server.Broadcast($"{config.ChatPrefix}: {message.Message}");
                     break;
-            }
-        }
 
-        /// <summary>
-        /// Broadcast a message to all connected clients
-        /// </summary>
-        /// <param name="message"></param>
-        public void SendMessage(RemoteMessage message)
-        {
-            if (message == null || server == null || !server.IsListening || listener == null) return;
+                case "players":
+                    var format = "ID: {0};NAME: {1};IP: {2};HP: {3}/{4};PING: {5};LANG: {6}";
```

Comment:
Also that format is just for consoles to translate into a player list. It's not actually for end users. Code wise I think it can be easily translated